### PR TITLE
Make server actions `maxDuration` timeout more clear in documentation

### PR DESCRIPTION
**Pull request by rocketman-21**

### CODE REVIEW

File: docs/02-app/02-api-reference/02-file-conventions/route-segment-config.mdx [hunk 1]
```diff
@@ -215,6 +215,7 @@ export const maxDuration = 5
 > **Good to know**:
 >
 > - If a `maxDuration` is not specified, the default value is dependent on your deployment platform and plan.
+> - If using [server actions](/docs/app/building-your-application/data-fetching/server-actions-and-mutations), set the `maxDuration` at the page level to change the default timeout of all server actions used on the page.
```
Comment (leerob) [inline docs/02-app/02-api-reference/02-file-conventions/route-segment-config.mdx:218]:
```suggestion
> - If using [Server Actions](/docs/app/building-your-application/data-fetching/server-actions-and-mutations), set the `maxDuration` at the page level to change the default timeout of all Server Actions used on the page. 
```

**Review by leerob — APPROVED** [review]

**Comment by leerob** [comment]
Could you run Prettier here please?

**Comment by rocketman-21** [comment]
sorry for the delay should be good @leerob

**Review by leerob — APPROVED** [review]

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: 30e90bc7804315c973a278ac4f4a0057e23b5e60

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: 30e90bc7804315c973a278ac4f4a0057e23b5e60

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: 30e90bc7804315c973a278ac4f4a0057e23b5e60

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

**Pull request merged**